src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,23 @@
+package calculator;
+
+public class StringCalculator {
+
+    private static String INPUT_STRING_DELIMETER = " ";
+
+    public static Number calculate(String input) {
+        if (input == null || input.isEmpty()) {
+            throw new IllegalArgumentException("잘못된 input입니다. (null or empty");
+        }
```

Comment:
```suggestion
        if (Objects.isNull(input) || input.isEmpty()) {
            throw new IllegalArgumentException("잘못된 input입니다. (null or empty");
        }
```
Objects 클래스를 활용하면 가독성을 더욱 높일수 있어요 !